src/tests/Currency.test.js

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
- [ ] Just as the previous reviewer suggested, please avoid disabling linter checks. The linters are there to help you adhere to good coding standards and ensure that the whole team is consistent with the same standards as well.
Kindly ensure that you remove and fix every other occurrence where you have disabled linters 👍